src/app/public/modules/colorpicker/colorpicker.component.ts

```diff
@@ -277,6 +335,20 @@ export class SkyColorpickerComponent implements OnInit, OnDestroy {
       this.hsva.saturation * this.sliderDimMax.saturation - 8,
       (1 - this.hsva.value) * this.sliderDimMax.value - 8,
       this.hsva.alpha * this.sliderDimMax.alpha - 8);
+
+    this.backgroundColorForDisplay = this.selectedColor.rgbaText;
```

Comment:
In the current demo, the color of the trigger button doesn't get changed until the user clicks `Apply`. That is no longer the case, as it changes to be in parallel with the changes you make while the picker is open. Is that deliberate?

Reply:
Good catch.